.github/workflows/markdown-altimetry.yml

```diff
@@ -0,0 +1,51 @@
+name: notebook_to_md (altimetry)
+
+on:
+  # External webhook trigger
+  repository_dispatch:
+   types: [push-notebook]
+  
+
+jobs:
+  commit-notebooks:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash -l {0}
+    steps:
+      - name: Checkout site
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive # Fetch Hugo themes and all extra projects
+          fetch-depth: 0 # Fetch all history for .GitInfo and .Lastmod
+          path: site
+
+      - name: Checkout coast
+        uses: actions/checkout@v3
+        with:
+          repository: British-Oceanographic-Data-Centre/COAsT
+          ref: develop
+          path: external
+
+      - name: Prepare environement
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          activate-environment: coast
+          environment-file: external/environment.yml
+          python-version: 3.9.13
+      - name: prep exec notebooks
+        run: |
+          conda info
+          conda list
+          conda config --show
+          cd $GITHUB_WORKSPACE/external
+          sudo apt install wget unzip -y
+          wget -q https://linkedsystems.uk/erddap/files/COAsT_example_files/COAsT_example_files.zip
+          unzip COAsT_example_files.zip && mv COAsT_example_files ./example_scripts/notebook_tutorials/runnable_notebooks/altimetry/example_files
+          ls /home/runner/work/COAsT-site/COAsT-site/external/example_scripts/notebook_tutorials/runnable_notebooks/altimetry/example_files
+          mkdir -p ./example_scripts/notebook_tutorials/markdown/altimetry
+          ls ./example_scripts/notebooks
```

Comment:
Just  a reminder to update this path to `./example_scripts/notebook_tutorials` for each of the workflows :)